packages/strapi-design-system/src/Portal/Portal.tsx

```diff
@@ -2,29 +2,16 @@ import * as React from 'react';
 
 import { createPortal } from 'react-dom';
 
-export interface PortalProps {
-  children: React.ReactNode;
-}
-
-export const Portal = ({ children }: PortalProps) => {
-  const rootRef = React.useRef<HTMLDivElement>(null!);
-  const [mounted, setMounted] = React.useState(false);
-
-  React.useLayoutEffect(() => {
-    rootRef.current = document.createElement('div');
-    rootRef.current.setAttribute('data-react-portal', 'true');
-    document.body.appendChild(rootRef.current);
+import { Box, BoxProps } from '../Box';
 
-    setMounted(true);
-
-    return () => {
-      rootRef.current?.remove();
-    };
-  }, []);
+export interface PortalProps extends BoxProps<HTMLDivElement> {
+  container?: HTMLElement | null;
```

Comment:
Does `null` make sense here or would it be better to allow only undefined or HTMLElement?

Reply:
I think it's fine to be open, the full type is `HTMLElement | undefined | null` in the sense you can omit the prop completely, so you don't have to pass `null` if you just want to use the default